tests/VirtoCommerce.Platform.Tests/Validators/BlobFolderValidatorTests.cs

```diff
@@ -0,0 +1,158 @@
+using AutoFixture;
+using FluentAssertions;
+using VirtoCommerce.Platform.Core.Assets;
+using VirtoCommerce.Platform.Web.Validators;
+using Xunit;
+
+namespace VirtoCommerce.Platform.Tests.Validators
+{
+    public class BlobFolderValidatorTests : ValidatorTestsHelper
```

Comment:
IMHO all these codes can be simplified by using Data-driven techniques were to one single method is received all test data via [Theory] attributes.